.github/workflows/integration.yml

```diff
@@ -30,7 +30,7 @@ env:
   S3_OLAP_ARN: ${{ vars.S3_OLAP_ARN }}
   S3_MRAP_ARN: ${{ vars.S3_MRAP_ARN }}
   KMS_TEST_KEY_ID: ${{ vars.KMS_TEST_KEY_ID }}
-  RUST_FEATURES: fuse_tests,s3_tests,fips_tests
+  RUST_FEATURES: fuse_tests,s3_tests,fips_tests,sse_kms
```

Comment:
Why add it back?

Reply:
something went wrong with rebasing, sorry